### ISSUE TITLE
Use the same version of unity, a lot of bugs

### ISSUE THREAD

**Issue by jiajixiang** [liuhaopen/UnityMMO#15]
Use the same version of unity, a lot of bugs
![QQ截图20191011164331](https://user-images.githubusercontent.com/28760037/66637790-5639b880-ec46-11e9-9a11-5c0c69a35df9.png)

**Comment by jiajixiang** [comment]
The number is really too much, I tried to solve part of it.
for example ：get，set. using
But because I am not familiar enough with your code, it may not be completely solved in a short time.
So many bugs，what is the reason?

**Comment by liuhaopen** [comment]
Entities does not exist? you can check it in menu Window->Packages Manager. or maybe you need to upgrade unity to at least 2019.2.0f1.

**Comment by jiajixiang** [comment]
> Entities does not exist? you can check it in menu Window->Packages Manager. or maybe you need to upgrade unity to at least 2019.2.0f1.


Yes, it is the problem of the package. The network download is very slow, and it is solved after the VPN is mounted.
